src/stepFunctions/commands/visualizeStateMachine/abstractAslVisualizationManager.ts

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+import { ConstructNode } from '../../../cdk/explorer/nodes/constructNode'
+
+import { Logger } from '../../../shared/logger'
+
+const localize = nls.loadMessageBundle()
+
+export abstract class AbstractAslVisualizationManager {
+    protected readonly name: string = 'AbstractAslVisualizationManager'
```

Comment:
I think you're always overwriting this when extending the class. Change to `private abstract name: string` with no initialization. That way, you're forced to give each extended class a name.